project/gh-action/infrastructure/kubernetes/service.yaml

```
@@ -0,0 +1,13 @@
+kind: Service
+metadata:
+  name: web-service
+spec:
+  type: NodePort 
+  ports:
+    - port: "80" 
+      nodePort: 30080
+      targetPort: wrong-port 
+      protocol: UDP 
+  selector:
+    app: frontend  
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)